modules/FormModel.js

```diff
@@ -676,74 +961,109 @@
+          }
+        } else {
+          var modelConfig = this.__createModelConfig(alias, config.mapping);
+          if (modelConfig) {
+            modelConfigs.push(modelConfig);
+          }
+        }
+      }, this);
       return modelConfigs;
     },
 
     /**
-     * A wrapper function that can invoke the pull callback on a Computed Configuration. The pull callback in the Computed
-     * Configuration will be passed a list of arguments. This list will be a copy of all the fields defined by the models array in
-     * order that they were defined including the models array order. If any model configuration does not contain a fields array, a
-     * hash will be provided for that entry into the model array that will contain a copy of all the attributes of that model.
+     * A wrapper function that can invoke the pull callback on a computed mapping during an event callback.
+     * Because an event callback predetermines the argument list, this method assumes the necessary computed configuration is
+     * bound as the part of the function context.
+     * When invoking the pull callback, it will pass in a object map from model alias to shallow copy of the tracked fields the
+     * computed value uses. It is NOT just the model, but a  copy of its attributes - feel free to change the properties.
+     * Example:
+     * fooBar: {
+     *   myModel: 'foo bar',
+     *   pull: function(models) {
+     *     console.log(models.myModel.foo, models.myModel.bar)
+     *   }
+     * }
+     * If any model mapping is tracking all fields by passing true as its config, a copy of all the attributes for that model will be provided.
      * @param [model] {Backbone.Model} the model that was updated. If provided, the cache will be updated
      * NOTE: requires the context of this function to be:
      * {
      *  formModel: <this form model>,
-     *  models: <the 'models' array of model configurations from the Computed Configuration>,
-     *  update: <the update callback from the Computed Configuration>,
+     *  alias: <the computed alias>,
      * }
      * @private
      * @method __invokeComputedPull
      */
     __invokeComputedPull: function(model) {
-      var args = [];
       if (model) {
         this.formModel.__updateCache(model);
       }
-      (function(formModel, pullCallback, modelConfigs) {
-        _.each(modelConfigs, function(modelConfig) {
-          if (modelConfig.fields) {
-            _.each(modelConfig.fields, function(field) {
-              args.push(formModel.__cloneVal(modelConfig.model.get(field)));
-            });
+      (function(formModel, alias) {
+        var hasAllModels = true,
+          config = formModel.getMapping(alias),
+          modelAliases = formModel.__getModelAliases(alias),
+          models = {};
+        if (!config.mapping.pull) {
+          console.log('Not pulling the computed: ' + alias + ', because no pull method was defined for this computed.');
```

Comment:
i imagine that the version of jquery we are using as a hard dependency requires a version of IE that has console.log...